Libraries/Inspector/InspectorPanel.js

```diff
@@ -84,19 +96,16 @@ class InspectorPanel extends React.Component<$FlowFixMeProps> {
 }
 
 InspectorPanel.propTypes = {
-  devtoolsIsOpen: PropTypes.bool,
-  inspecting: PropTypes.bool,
-  setInspecting: PropTypes.func,
   inspected: PropTypes.object,
-  perfing: PropTypes.bool,
-  setPerfing: PropTypes.func,
-  touchTargeting: PropTypes.bool,
-  setTouchTargeting: PropTypes.func,
-  networking: PropTypes.bool,
-  setNetworking: PropTypes.func,
 };
 
-class Button extends React.Component<$FlowFixMeProps> {
+type InspectorPanelButtonProps = $ReadOnly<{|
+  onClick?: ?Function,
```

Comment:
You're right. This should also take the same signature:

```tyepscript
onClick?: ?(val: boolean) => void
```

Reply:
Yup, we should go ahead and update this to `?(val: boolean) => void`.

Reply:
Now that I think about it more, a lot of these types are unnecessarily optional. For example, I think `onClick` should not be optional, since we assume (in the implementation of this component) that it's a function. Do you mind making these optional properties, that don't need to be optional, non-optional?